projects/storefrontlib/src/cms-structure/outlet/outlet.providers.ts

```diff
@@ -0,0 +1,41 @@
+import { InjectionToken, StaticProvider, Type } from '@angular/core';
+import { OutletPosition } from './outlet.model';
+
+/**
+ * @private We plan to drive the outlets by standard configuration
+ */
+export const PROVIDE_OUTLET_OPTIONS = new InjectionToken<ProvideOutletOptions>(
+  'Options for providing the outlet'
```

Comment:
I assume this increasing bundle size unnecessary, perhaps we shorten this a bit

Reply:
Nice catch. Shortened to `PROVIDE_OUTLET_OPTIONS`